src/modules/react-reconciler/Atlas.tsx

```diff
@@ -517,6 +517,7 @@ export const Atlas: React.FC<
         .atlas-canvas { flex: 1 1 0px; }
         .atlas-canvas:focus, .atlas-static-container:focus { outline: none }
         .atlas-canvas:focus-visible, .atlas-canvas-container:focus-visible { outline: var(--atlas-focus, 2px solid darkorange) }
+        .atlas-static-preset { touch-action: inherit; }
```

Comment:
I do like this pattern for the presets - I'm not a huge fan of this inline style at the moment, but not sure of what else to do (aside from setting directly on elements which would be even more difficult to override).